appveyor.yml

```diff
@@ -6,6 +6,8 @@ environment:
     - nodejs_version: "10"
 
 cache:
+  - node_modules
+  - _target
```

Comment:
We can't cache `deltachat-core-rust/target` folder since git will complain when doing `git submodule update --init` because the folder is not empty. So instead we cache a temporary `_target` folder which we later move into `deltachat-core-rust`.